compiler/damlc/daml-stdlib-src/DA/Upgrade.daml

```diff
@@ -12,6 +12,8 @@ module DA.Upgrade
   , Convertible(..)
   , GenConvertible
   , Upgrade(..)
+  , UpgradeInstance(..)
```

Comment:
Another fun example for @hurryabit where we leak the `*Instance` classes :slightly_smiling_face: